### DISCUSSION TITLE
Read Telemetry via API

### DISCUSSION THREAD

**Discussion by baljeetsingh** [Sunbird-Obsrv/Community#43] [Documentation]
### Is there a way to read telemetry via API ?

**To start with,**
We would like to get data for Content Access like when user is starting to read content (video, document, etc)

Additional context, general LMS system shows the usage access for user on when and what all content was accessed by user.

**FYI / Broader use case**
Now when there are users, students, and say executives who are mandated to go through courses and attempt assessment but then we do not how much time was spent by user on content before attempting assessment, and if user did use content before re-attempt of assessment .

So building a csv using telemetry to show when all the content was accessed by user.

**Comment by anandp504** [comment]
@baljeetsingh The telemetry gets backed up to the cloud storage and also gets ingested into Druid analytics data store. You should be able to query from Druid using the Druid Native Query APIs. For more information, please refer to the Druid documentation on [Native queries](https://druid.apache.org/docs/latest/querying/querying.html).
